src/ripple/app/misc/HashRouter.h

```diff
@@ -159,12 +160,10 @@ class HashRouter
     std::mutex mutable mMutex;
 
     // Stores all suppressed hashes and their expiration time
-    hash_map <uint256, Entry> mSuppressionMap;
+    beast::aged_unordered_map<uint256, Entry, clock_type,
```

Comment:
Can't `aged_unordered_map` be instantiated with `Stopwatch` instead of `Stopwatch::clock_type`?